keps/sig-scheduling/3902-decoupled-taint-manager/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: Decouple TaintManager from NodeLifeCycleController 
+kep-number: 3902
+authors:
+  - "@yuanchen8911"
+  - "@atosatto"
+  - "@ddebroy"
+  - "@ravisantoshgudimetla"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-apps
+  - sig-node
+status: implementable
+creation-date: 2023-03-07
+reviewers:
+  - "@Huang-Wei" 
+  - "@alculquicondor"
+approvers:
```

Comment:
you can put me for sig node reviewer of approver

Reply:
Added your name to reviewers.